pyproject.toml

```diff
@@ -153,28 +153,30 @@ skip-string-normalization = 1
 
 [tool.ruff]
 line-length = 88
```

Comment:
If we are removing `exclude`, we might as well remove `line-length` since it is also redundant.